backend/swagger-output.json

```diff
@@ -831,7 +888,7 @@
       "post": {
         "tags": ["Réservations"],
         "summary": "Créer une réservation",
-        "description": "Nécessite d'être de niveau 2",
```

Comment:
Why did you delete this line ? If you didn't wanted to change this file you can remove it from this PR